examples/graphql-react-apollo/src/index.js

```diff
@@ -3,17 +3,19 @@ import ReactDOM from 'react-dom'
 import { ApolloProvider } from '@apollo/react-hooks'
 import { client } from './ApolloClient'
 import { LoginForm } from './LoginForm'
+import { User } from './User'
 
 // Start the mocking conditionally.
 if (process.env.NODE_ENV === 'development') {
   const { worker } = require('./mocks/browser')
   worker.start()
 }
 
+const url = window.location.href
 ReactDOM.render(
   <React.StrictMode>
     <ApolloProvider client={client}>
-      <LoginForm />
+      {url.includes('/user') ? <User /> : <LoginForm />}
```

Comment:
This is a really hacky way to do paging, but I didn't want to introduce another dependency in this PR 😅

If you would prefer to show an example with `react-router-dom`, I'm glad to update this 👍